l1-contracts/contracts/bridge/L1SharedBridge.sol

```diff
@@ -207,7 +244,7 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
     }
 
     /// @dev Initializes the l2Bridge address by governance for a specific chain.
-    function initializeChainGovernance(uint256 _chainId, address _l2BridgeAddress) external onlyOwner {
+    function initializeChainGovernance(uint256 _chainId, address _l2BridgeAddress) external onlyOwnerOrAdmin {
```

Comment:
Admin shouldn't be able to overwrite the bridge. I would rather make two functions:
- `initializeChainGovernance` onlyOwnerOrAdmin (check that `chainId`)
- `reinitializeChainGovernance` onlyOwner

Reply:
When will we need to overwrite the bridge? Or just in case?